### ISSUE TITLE
Gallery sample app vertical orientation not working

### ISSUE THREAD

**Issue by weijie** [bumptech/glide#602]
I am new to glide,first of all thanks for creating such a nice library.

While playing around with the sample gallery app, I tried to change the orientation from horizontal to vertical but the app shows blank screen. What would it take to implement the vertical version?
Thank you!

**Comment by TWiStErRob** [comment]
Do you mean this one?
https://github.com/bumptech/glide/tree/master/samples/gallery 
Note that it's on the master branch, which is not stable yet. You may want to look around on 3.0 branch (latest 3.6.1) unless you are an early adopter for Glide 4.0.

In any case, thanks for the report, rotation should really just work in any sample. We'll take a look.

**Comment by weijie** [comment]
Yes, that's the one and to be clear, rotation works fine. What I did was that I changed the following line 
HorizontalGalleryFragment.java line 39: layoutManager.setOrientation(RecyclerView.HORIZONTAL); to layoutManager.setOrientation(RecyclerView.VERTICAL); and it shows blanks screen after that so I was wondering what needs to be modified to be able to set to RecyclerView.VERTICAL.

To give you some background, I was looking for examples (good practices) of using recyclerview and glide but I didn't find any sample in the 3.0 branch. Gallery sample in 4.0 is exactly what I wanted expect that I want to be able to scrolling vertically.
